src/components/LoginPage/Slide3.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import arrow from 'components/LoginPage/images/arrow.svg'
+import slideImage1 from 'components/LoginPage/images/slide-3_img-1.svg';
+import slideImage2 from 'components/LoginPage/images/slide-3_img-2.svg';
+
+const Slide = styled.section`
+  padding-bottom: 0%;
+  text-align: center;
+  .note-example {
+    width: 61%;
+  }
+   .edit-example {
+    position: relative;
+    margin-top: -9%;
+  }
+  .bottomPosition {
+    margin-top: -65%;
+    margin-bottom: 59%;
+    position: relative;
+  }
+  .bottomTooltip {
+    width: 22%;
+    padding-top: 3%;
+    padding-bottom: 3%;
+  }
+  .bottomHeader {
+    margin-top: -53%;
+    margin-bottom: 45%;
+    position: relative;
+  }
+  .shadow {
```

Comment:
Shadow for what?
Also, it's duplicated in `Slide1.js` and `Slide3.js`

Reply:
This is shadow for svg images. Unfortunately, when this svg images imports from Figma, they have not shadows...

Reply:
We can add it to the `Img` component then. To use like
```
<Img shadow="true">
```
Right now it's not clear to which component is it going to be applied, because it's just ".shadow".

Reply:
Solved in commit [607a206](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/607a2060c5163550196d37c8b73b544063ea1bfc)